PlayCover/AppInstaller/Installer.swift

```diff
@@ -36,13 +36,15 @@ class Installer {
     }
 
     // swiftlint:disable function_body_length
-    static func install(ipaUrl: URL, export: Bool, returnCompletion: @escaping (URL?) -> Void) {
+    static func install(ipaUrl: URL, export: Bool, ipaSourcePTD: Bool?, returnCompletion: @escaping (URL?) -> Void) {
         // If (the option key is held or the install playtools popup settings is true) and its not an export,
         //    then show the installer dialog
         let installPlayTools: Bool
 
         if (Installer.isOptionKeyHeld || InstallPreferences.shared.showInstallPopup) && !export {
             installPlayTools = installPlayToolsPopup()
+        } else if !(ipaSourcePTD ?? true) {
```

Comment:
It would be better to ensure `ipaSourcePTD` to not be null before evaluating it instead of giving a default operator of `true`. Currently, this would make the last else statement unable to run if the app is installed via default means, and making the app always install without PlayTools despite the user's settings.

Reply:
I have tested that, it does not do that. last else statement works fine because in source if you define `true` or in other installers like app library `nil` it will be skipped to `else` part, only works while it's `false` from source.